modules/javafx.graphics/src/main/java/com/sun/javafx/css/BitSet.java

```diff
@@ -229,10 +230,14 @@ public boolean contains(Object o) {
     /** {@inheritDoc} */
     @Override
     public boolean containsAll(Collection<?> c) {
+        if (this.getClass() != c.getClass()) {
```

Comment:
this change is not equivalent: in the old code, null `c` would return false, in the new it'll throw an NPE

Reply:
That's correct, before it broke the contract of `Set`.  It's not supposed to return `false` when the parameter is `null`.  This class is sometimes exposed as public API (via `ObservableSet`) and so must respect the `Set` contract.